src/desktop/open_uri.rs

```diff
@@ -57,14 +57,14 @@ use zbus::zvariant::{Fd, SerializeDict, Type};
 use super::{HandleToken, DESTINATION, PATH};
 use crate::{
     helpers::{call_basic_response_method, session_connection},
-    Error, WindowIdentifier,
+    ActivationToken, Error, WindowIdentifier,
 };
 
 #[derive(SerializeDict, Type, Debug, Default)]
 #[zvariant(signature = "dict")]
 struct OpenDirOptions {
     handle_token: HandleToken,
-    activation_token: Option<String>,
+    activation_token: ActivationToken,
```

Comment:
It is optional so why make it no longer the case?

Reply:
There is ActivationToken::None which is the default so wrapping in option seems redundant.

Reply:
Sure, but it is simpler to take `Impl<Into<Option<T>>` that way you don't have to handle both cases in your app side :) see all the other options.